ui/src/app/pages/login/account-creation/account-creation-institution.spec.tsx

```diff
@@ -1,417 +1,332 @@
+      }
+      expect(screen.getByText(name)).toBeInTheDocument();
+    }
+
+    await userEvent.click(
+      within(
+        screen.getByDisplayValue(
+          'Research fellow (a post-doctoral fellow or medical resident in training)'
+        ).parentElement.parentElement
+      ).getByRole('button')
+    );
+
+    // Check institution Role drop down lists
+    const institutionRoleOpt = AccountCreationOptions.institutionalRoleOptions;
+    for (let index = 0; index < institutionRoleOpt.length; index++) {
+      if (institutionRoleOpt[index].value !== InstitutionalRole.FELLOW) {
+        expect(
+          await screen.findByText(institutionRoleOpt[index].label)
+        ).toBeInTheDocument();
+      } else {
+        expect(
+          screen.getByDisplayValue(
+            'Research fellow (a post-doctoral fellow or medical resident in training)'
+          )
+        ).toBeInTheDocument();
+      }
+    }
   });
 
-  // Email address is entered, but the input hasn't been blurred. The form should know that a
-  // response is required, but the API request hasn't been sent and returned yet.
-  expect(getInstance(wrapper).validate().checkEmailResponse).toContain(
-    'Institutional membership check has not completed'
-  );
-
-  // Once we blur the input, the API request is sent. Since asdf.com is not a member, it will
-  // block form submission.
-  getEmailInput(wrapper).simulate('blur');
-
-  await waitOneTickAndUpdate(wrapper);
-  expect(getInstance(wrapper).validate().checkEmailResponse).toContain(
-    'Email address is not a member of the selected institution'
-  );
-
-  expect(getEmailErrorMessage(wrapper).getDOMNode().textContent).toBe(
-    'Your email does not match your institution'
-  );
-});
-
-it('should display validation icon only after email verification', async () => {
-  const wrapper = component();
-  await waitOneTickAndUpdate(wrapper);
-
-  // Choose 'VUMC' and enter an email address.
-  getInstitutionDropdown(wrapper).props.onChange({
-    ...eventDefaults,
-    originalEvent: undefined,
-    value: 'VUMC',
-    target: { name: '', id: '', value: 'VUMC' },
+  it('should show user-facing error message on data load error', async () => {
+    // Mock institution detail api call to return error
+    const mockGetPublicInstitutionDetails = jest.spyOn(
+      institutionApi(),
+      'getPublicInstitutionDetails'
+    );
+    mockGetPublicInstitutionDetails.mockRejectedValueOnce(
+      new Response(null, { status: 500 })
+    );
+    const { container } = setup();
+
+    await waitForSpinnerToGoAway(container);
+
+    // Confirm it shows error after spinner goes away
+    expect(
+      screen.getByText(
+        /an error occurred loading the institution list\. please try again or contact\./i
+      )
+    ).toBeInTheDocument();
   });
-  getEmailInput(wrapper).simulate('change', {
-    target: { value: 'asdf@wrongDomain.com' },
+
+  it('should reset role value & options when institution is selected', async () => {
```

Comment:
Do we still want role to be cleared out when you select an institution? It seems like since all institutions have the same roles, these shouldn't necessarily be connected.